tests/mmm/test_delayed_saturated_mmm.py

```diff
@@ -550,18 +556,18 @@ def mock_property(self):
             None,
             {
                 "intercept": {"dist": "Normal", "kwargs": {"mu": 0, "sigma": 2}},
-                "beta_channel": {
+                "saturation_beta": {
                     "dist": "HalfNormal",
                     "kwargs": {"sigma": np.array([0.4533017, 0.25488063])},
                 },
-                "alpha": {
+                "adstock_alpha": {
                     "dist": "Beta",
                     "kwargs": {
                         "alpha": np.array([3, 3]),
                         "beta": np.array([3.55001301, 2.87092431]),
                     },
                 },
-                "lam": {
+                "saturation_lambda": {
```

Comment:
This will also be  `saturation_lam` since the parameter name is `lam` to avoid the lambda keyword